MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js

```diff
@@ -74,19 +83,21 @@ define([
         saveImageDetailsAction: function () {
             var saveDetailsUrl = this.mediaGalleryEditDetails().saveDetailsUrl,
                 modalElement = $(this.modalSelector),
-                dataForm = modalElement.find('#image-edit-details-form'),
-                imageId = this.imageModel().getSelected().id;
-
-            if (dataForm.validation('isValid')) {
```

Comment:
I believe this validation has to stay, it probably came through when you pull the updates, let's make sure this still happens.